index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="css/style.css">
    <title> Mahatma Gandhi </title>
```
please ensure to add title in the header section for better SEO optimization